### ISSUE TITLE
Is it possible to add link in menu item?

### ISSUE THREAD

**Issue by HW-Siew** [alejandropascual/quick-restaurant-menu#3]
Hi, is it possible to add link in menu item? So that, for example, title or image is clickable?

**Comment by alejandropascual** [comment]
Yes, with the PRO version image is clickable, shows a popup with additional information.
